robotbase/src/main/java/frc/robot/commands/Autos.java

```diff
@@ -5,23 +5,12 @@
 package frc.robot.commands;
```

Comment:
Looks like we're not using this file anywhere. Let's delete it (unless GitHub is wrong and we are using it)